### PR TITLE
Update Clippy Flow responses

### PR DESCRIPTION
**Pull request by Hugoyhu**
Update active channel list, #cooking is archived, many others like #jenga and #film are rarely active, modify clippy flow messages.

### CODE REVIEW

File: flows/default.js [hunk 1]
```diff
@@ -203,7 +203,7 @@ const loadFlow = (app) => {
...
         await sendMessage(app, body.event.channel, `Try clicking the red :ping: on your sidebar to the left :eyes:`)
-        await sendCustomizedMessage(app, body.event.channel, `<@${body.event.user}> As I was saying before I got distracted, we have _hundreds_ of these "channels" in the community, covering every topic you can think of, from \`#gamedev\` and \`#code\` to \`#photography\` and \`#cooking\`. We have nearly 1,000 weekly active members on here—wowee, that's a lot!!!`, 'https://cloud-8iwducj5z.vercel.app/moshed-2020-9-8-13-47-23.jpg', null, 10000)
+        await sendCustomizedMessage(app, body.event.channel, `<@${body.event.user}> As I was saying before I got distracted, we have _hundreds_ of these "channels" in the community, covering every topic you can think of, from \`#gamedev\` and \`#code\` to \`#photography\` and \`#ship\`. We have over 700 weekly active members on here—wowee, that's a lot!!! Oh, and we have over 1000 active channels! :smiley:`, 'https://cloud-8iwducj5z.vercel.app/moshed-2020-9-8-13-47-23.jpg', null, 10000)
```
Comment (sampoder) [inline flows/default.js:206]: Could we not go to #ship, but more like a niche channel, you know?

Comment (Hugoyhu) [reply flows/default.js:206]: Oh you mean like a "chat" channel, more?

File: flows/default.js [hunk 2]
```diff
@@ -257,7 +257,7 @@ const loadFlow = (app) => {
     await sendCustomizedMessage(app, body.channel.id, `‎‏‏‎ ‎`, 'https://cloud-603yzf4nn.vercel.app/screen_shot_2020-09-08_at_2.36.29_pm.png', ' ‎', 1000)
     await sendMessage(app, body.channel.id, `‎‏‏‎...`, 1000)
     await sendMessage(app, body.channel.id, `‎‏‏‎...`, 1000)
-    await sendMessage(app, body.channel.id, `Before you proceed, please make sure to read and abide by our <https://hackclub.com/conduct|code of conduct>. Every community member is expected to follow the code of conduct anywhere in the community.`)
+    await sendMessage(app, body.channel.id, `Before you proceed, please make sure to read and abide by our <https://hackclub.com/conduct|code of conduct>. Every community member is expected to follow the code of conduct anywhere in the community, including in events and meetings. .`)
```
Comment (sampoder) [inline flows/default.js:260]:
```suggestion
    await sendMessage(app, body.channel.id, `Before you proceed, please make sure to read and abide by our <https://hackclub.com/conduct|code of conduct>. Every community member is expected to follow the code of conduct anywhere in the community, including in events and video calls. .`)
```

Comment (Hugoyhu) [reply flows/default.js:260]: Oh I meant club meetings lol

File: flows/default.js [hunk 3]
```diff
@@ -300,9 +300,10 @@ const loadFlow = (app) => {
       <#CBX54ACPJ> - Share your photos!
       <#CC78UKWAC> - :jenga_sleep:
       <#C8P6DHA3W> - Don't enter if you're hungry!
-      <#C010SJJH1PT> - Learn about cooking!
+      <#C0EA9S0A0> - For general coding! There's also <#C0131FX5K98>, <#C016G8A6MKJ>, <#C012LPZUAPR>, <#C019NDLMJCV>, <#C0120KJH1SB>, <#C0121LVV79P>, and <#CLWA83V7Y>, to name a few.
```
Comment (sampoder) [inline flows/default.js:303]: maybe we should keep the list format, tho I'll leave that up to Matthew

File: flows/default.js (hunk 1 above)
Comment (sampoder) [reply flows/default.js:206]: kinda, but a non technical one, a human one

File: flows/default.js (hunk 3 above)
Comment (Hugoyhu) [reply flows/default.js:303]: Lol K

File: flows/default.js (hunk 2 above)
Comment (sampoder) [reply flows/default.js:260]: ahhh

Comment (sampoder) [reply flows/default.js:260]: check now?

Comment (Hugoyhu) [reply flows/default.js:260]: Looks good

File: flows/default.js (hunk 1 above)
Comment (Hugoyhu) [reply flows/default.js:206]: Gotcha, I"ll get on it tomorrow(oops, in the morning today)

**Comment by sampoder** [comment]
Closing due to merge conflicts :)

**Pull request closed without merging**